templates/authorizer.py

```diff
@@ -12,6 +13,7 @@
 import cfnutils.kms
 import cfnutils.output
 
+from custom_resources.cognito import UserPoolClient
```

Comment:
Not wrong, but I prefer to use fully qualified names:
> import custom_resources.cognito
and below
> add_resource(custom_resources.cognito.UserPoolClient())

to stress the fact that this is a *custom* UserPoolClient, not the default one

Reply:
Will update.